### DISCUSSION TITLE
what to do with drive/volume after it fills up

### DISCUSSION THREAD

**Discussion by esembeit** [Chia-Network/chia-blockchain#2959] [Q&A]
Hi Community,

New to Chia, sorry if this is a trivial question. I have been reading and searching for a while now and can seem to find the explanation. But what do you do with a drive once it fills up. I under the temp, temp2 and final directory. I also see everywhere that says raid isnt needed. So when you fill up a drive what do you do with it then. i understand you can tell chia to change final destination or swap out drive but then what. do you only need the created plots for a limited time and then they can be overwritten or is there a service on a full node that keeps track of all volumes with plots on them. then you just keep filling up volumes one after another but another service keeps them around. can someone share with a newb on what happens next. thanks

**Comment by SackofFluor** [comment]
The drives have to stay connected to the machine.
So using a USB-HUb and attaching external harddrives to them works well. If you don't want to use so many external drives, there are bigger casings that can accommodate many drives and are just connected by USB.

Best, SackofFluor
